spec/lib/common/exceptions/detailed_schema_errors_spec.rb

```diff
@@ -130,11 +131,20 @@
   end
 
   context 'patterns' do
+    it 'has title, detail, and meta' do
+      data['pattern'] = Faker::Lorem.sentence
+      expect(subject[:title]).to eq 'Invalid format'
+      expect(subject[:detail]).to eq "'#{data['pattern']}' did not match the defined format"
+      expect(subject[:meta][:regex]).to eq '.@.'
```

Comment:
Is this regex from the draft-7? This is what we had earlier, isn't it?

Reply:
Yeah, the diff is being strange. It's what we had before.